CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## v1.12.0 (03.03.2021)
+
+### Enhancements
+
+#### :gear: Server-Side Web Assets Caching ([testcafe-hammerhead/#863](https://github.com/DevExpress/testcafe-hammerhead/issues/863))
+
+TestCafe's reverse-proxy can now cache web assets (like images, scripts and videos). When TestCafe revisits a website, it loads assets from this cache, saving time on repetetive network requests.
+
+To enable server-side caching, use any of the following:
```

Comment:
`use any of the following OPTIONS?`

Reply:
They are not all options. I couldn't find a common word, so I left it out

Reply:
Ok